kmod/toa/toa.c

```diff
@@ -49,10 +49,8 @@ struct toa_stats_entry toa_stats[] = {
 	TOA_STAT_ITEM("getname_toa_mismatch", GETNAME_TOA_MISMATCH_CNT),
 	TOA_STAT_ITEM("getname_toa_bypass", GETNAME_TOA_BYPASS_CNT),
 	TOA_STAT_ITEM("getname_toa_empty", GETNAME_TOA_EMPTY_CNT),
-#ifdef TOA_IPV6_ENABLE
```

Comment:
`TOA_IPV6_ENABLE` is intended for all IPv6 related codes. Its effective scope should be inclusive of both 6to6 and 6to4.